### ISSUE TITLE
Glur over UIViewController gives Yellow view with restriction sign

### ISSUE THREAD

**Issue by dollar2048** [joogps/Glur#17]
When I apply `.glur(...)` over the SwiftUI view which is `UIViewControllerRepresentable` it gives some Yellow view with a restriction sign:
<img width="217" alt="image" src="https://github.com/user-attachments/assets/f4fe2872-3d1b-48a3-bbf3-68169a45d4e7">
The code:
```swift
import Glur
import SwiftUI

struct GlurView: View {
    var body: some View {
        VCRepresentable()
            .glur(radius: 15, offset: 0.5, interpolation: 0.6, direction: .down)
    }
}

struct VCRepresentable: UIViewControllerRepresentable {
    func makeUIViewController(context: Context) -> UIViewController {
        let vc = UIViewController()
        vc.view.backgroundColor = .red
        return vc
    }

    func updateUIViewController(_ uiView: UIViewController, context: Context) {}
}

#Preview {
    GlurView()
}
```

What could be the reason? I would appreciate any hint.

Ultimately, I need `UIViewController` to present `AVPlayerViewController` with some Player inside.
Or do you have any idea how to implement that blur effect, like in this GitHub repository (without any frosty material background), over the video player?

**Comment by theedov** [comment]
Maybe this is the reason?

<img width="835" alt="Image" src="https://github.com/user-attachments/assets/461c6c46-1019-4f15-acf6-7addb760ad6b" />

**Comment by joogps** [comment]
> Maybe this is the reason?
> 
> <img alt="Image" width="835" src="https://private-user-images.githubusercontent.com/30866972/414946280-461c6c46-1019-4f15-acf6-7addb760ad6b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAwMjAxNDMsIm5iZiI6MTc0MDAxOTg0MywicGF0aCI6Ii8zMDg2Njk3Mi80MTQ5NDYyODAtNDYxYzZjNDYtMTAxOS00ZjE1LWFjZjYtN2FkZGI3NjBhZDZiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIwVDAyNTA0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzODdkNWM5ODFiMGRkMDhjMjI5Nzk5MWZiY2Y2OTdmNGYyNzAyNWI2YjlhZDVlMTgyMDNjZDY2MjVmMTI5OGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5ZmSnI2UUoMr7Y6_GPhmEVE8BJlpLhRy3Luhiqh2TxA">

Yes! Since I'm using Apple's SwiftUI shader API for Glur, UIKit-backed views are not supported.
